charts/kube-prometheus-stack/ci/05-ingress-and-gateway-routes-values.yaml

```diff
@@ -0,0 +1,17 @@
+alertmanager:
+  ingress:
+    enabled: true
+    hosts:
+      - "*.test.com"
+
+
+prometheus:
+  ingress:
+    enabled: true
+    hosts:
+      - "*.test.com"
+  route:
+    main:
+      enabled: true
```

Comment:
Does it make sense to have both ingress and gateway enabled?

Reply:
Would that not cause issues?

Reply:
Sure! For each value test file. helm-chart-test will do a full helm install. Increase the amount if ci/values.yaml would increase the CI times as well.

To avoid that, I merged distinct features into one values.yaml. It might be pointless to enabled ingress and routes, but I would like to have to feature under test coverage.

Reply:
No worries that works for me :)